### PR TITLE
Update Installation.md

### PR DESCRIPTION
**Pull request by Helias**
Add note about cmake command and $HOME variable for the cmake path.

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ Check the options here if you know what you're doing: [CMake options](CMake-opti
 
 At this point, you must be in your "build/" directory.
 
+**Note**: in the follows command the variable`$HOME` is the path of the **current user**, so if you are logged as root, $HOME will be "/root".
```
Comment (FrancescoBorzi) [inline docs/Installation.md]: missing space before `$HOME`

File: docs/Installation.md [hunk 2]
```diff
@@ -62,6 +64,8 @@ make install
 
 **CMake on Mac OS X:**
 
+**Note**: in the follows command the variable`$HOME` is the path of the **current user**, so if you are logged as root, $HOME will be "/root".
```
Comment (FrancescoBorzi) [inline docs/Installation.md]: missing space before `$HOME`

**Comment by Helias** [comment]
fixed spaces

File: docs/Installation.md [hunk 3]
```diff
@@ -49,6 +49,8 @@ Check the options here if you know what you're doing: [CMake options](CMake-opti
 
 At this point, you must be in your "build/" directory.
 
+**Note**: in the follows command the variable `$HOME` is the path of the **current user**, so if you are logged as root, $HOME will be "/root".
```
Comment (BarbzYHOOL) [inline docs/Installation.md]: in the following command, the environment variable $HOME points towards your current user home directory (/home/my_user or /root if you are logged as root). Try it `echo $HOME`.

@Helias

**Pull request merged**